.github/workflows/validation.yaml

```diff
@@ -66,3 +70,125 @@ jobs:
         with:
           junit_files: "**/target/*-reports/TEST*.xml"
           check_run_annotations: all tests, skipped tests
+  checkstyle:
+    needs: [check-permissions]
+    timeout-minutes: 5
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Remove format labels
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const { owner, repo } = context.repo;
+            const prNumber = context.payload.pull_request.number;
+            
+            // Remove the label if it exists
+            const existingLabels = await github.rest.issues.listLabelsOnIssue({
+              owner,
+              repo,
+              issue_number: prNumber,
+            });
+            const labelsToRemove = existingLabels.data.map(label => label.name).filter( label => label.startsWith("format:") );
+            for (const labelToRemove of labelsToRemove) {
+              await github.rest.issues.removeLabel({
```

Comment:
Is there also a .removeLabels operation, which directly takes all labels to remove, like for the .addLabels method?

Reply:
There's only remove one, remove all or replace all